include/PrimeTower/PrimeTowerInterleaved.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2024 Ultimaker B.V.
+// CuraEngine is released under the terms of the AGPLv3 or higher.
+
+#ifndef PRIME_TOWER_INTERLEAVED_H
+#define PRIME_TOWER_INTERLEAVED_H
+
+#include "PrimeTower/PrimeTower.h"
+
+namespace cura
+{
+
+class PrimeTowerInterleaved : public PrimeTower
```

Comment:
A bit of documentation on how this is different would be nice (mostly looking at the future here. We now know what the difference is, but future developers might not)